src/qt/askpassphrasedialog.cpp

```diff
@@ -111,8 +111,8 @@ void AskPassphraseDialog::accept()
             // Cannot encrypt with empty passphrase
             break;
         }
-        QMessageBox::StandardButton retval = QMessageBox::question(this, tr("Confirm wallet encryption"),
-                 tr("Warning: If you encrypt your wallet and lose your passphrase, you will <b>LOSE ALL OF YOUR BITCOINS</b>!") + "<br><br>" + tr("Are you sure you wish to encrypt your wallet?"),
+        QMessageBox::StandardButton retval = QMessageBox::question(this, tr("Confirm create encrypted wallet"),
+                 tr("Warning: If you encrypt your wallet and lose your passphrase, you will <b>LOSE ALL OF YOUR BITCOINS</b>!") + "<br><br>" + tr("Are you sure you want to create a new encrypted wallet?"),
```

Comment:
Could be cool to seize the opportunity to remove the `<b>` tag from tr() call.
See [this](https://github.com/bitcoin/bitcoin/blob/master/src/qt/sendcoinsdialog.cpp#L281) or [this](https://github.com/bitcoin/bitcoin/blob/master/src/qt/sendcoinsdialog.cpp#L356) for some examples.
(unless somebody knows of a good reason style tags should be kept for translators but I don't think so)